homeassistant/components/open_meteo/weather.py

```diff
@@ -78,3 +78,49 @@ def wind_bearing(self) -> float | str | None:
         if not self.coordinator.data.current_weather:
             return None
         return self.coordinator.data.current_weather.wind_direction
+
+    @property
+    def forecast(self) -> list[Forecast] | None:
+        """Return the forecast in native units."""
+        if self.coordinator.data.daily is None:
+            return None
+
+        forecasts: list[Forecast] = []
+        for index, time in enumerate(self.coordinator.data.daily.time):
+            forecast = Forecast(
+                datetime=time.isoformat(),
+            )
+
+            if self.coordinator.data.daily.weathercode is not None:
+                forecast["condition"] = WMO_TO_HA_CONDITION_MAP.get(
+                    self.coordinator.data.daily.weathercode[index]
+                )
+
+            if self.coordinator.data.daily.precipitation_sum is not None:
+                forecast[
+                    "precipitation"
+                ] = self.coordinator.data.daily.precipitation_sum[index]
+
+            if self.coordinator.data.daily.temperature_2m_max is not None:
+                forecast[
+                    "temperature"
```

Comment:
Should this be `tempmax` to match `templow`?

Reply:
There is no such thing...?